README.md

```diff
@@ -138,12 +170,12 @@ Please take a look at our [wiki] for development documentations!
 
 ## Acknowledgement
 
-  Some functionalities of DataPrep are inspired by the following packages.
+  Some functionalities of Dataprep are inspired by the following packages.
```

Comment:
Should this be DataPrep?

Reply:
You're right. I got confused because in the current Readme file in some sections is mentioned "Dataprep" and in others "DataPrep". I'll update the info.

Reply:
Done.